test/scripts/install-kind.sh

```diff
@@ -19,7 +19,7 @@ if [ "$KIND_IPV4_SUPPORT" == true ] && [ "$KIND_IPV6_SUPPORT" == true ]; then
   sudo curl -Lo /usr/local/bin/kind https://github.com/aojea/kind/releases/download/dualstack/kind
   sudo chmod +x /usr/local/bin/kind
 else
-  curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.10.0/kind-linux-amd64
+  curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.11.1/kind-linux-amd64
```

Comment:
this version works for dual stack too, if you check this `if` the `then` statement installs kind from my fork